libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalIndividualView/CausalIndividualChartStyles.ts

```diff
@@ -201,7 +201,7 @@ export const causalIndividualChartStyles: () => IProcessedStyleSet<ICausalIndivi
         paddingRight: "120px"
       },
       highchartContainer: {
-        width: "1100px"
+        width: "1300px"
```

Comment:
maybe this should be a global constant re-used in several places?  I see it in several files.

Reply:
This just a temp fix to solve style issue for new added highchart, fixed width will eventually be removed, it's different size based on different chart